internal/brokers/examplebroker/broker.go

```diff
@@ -220,50 +331,61 @@ func (b *Broker) GetAuthenticationModes(ctx context.Context, sessionID string, s
 
 		case "webview":
 			// This broker does not support webview
+
+		case "newpassword":
+			if layout["entry"] == "" {
+				break
+			}
+			allModes["mandatoryreset"] = map[string]string{
+				"selection_label": "Password reset",
+				"ui": mapToJSON(map[string]string{
+					"type":  "newpassword",
+					"label": "Enter your new password",
+					"entry": "chars_password",
+				}),
+			}
+
+			if layout["skip-button"] != "" {
+				allModes["optionalreset"] = map[string]string{
+					"selection_label": "Password reset",
+					"ui": mapToJSON(map[string]string{
+						"type":        "newpassword",
+						"label":       "Enter your new password",
```

Comment:
You can append something like: "3 days until mandatory" to illustrate the example.